### ISSUE TITLE
PHP Notice when viewing Customers graph

### ISSUE THREAD

**Issue by robincornett** [awesomemotive/easy-digital-downloads#9530]
### Expected behavior
I expect to not have PHP notices when viewing reports.

### Actual behavior
When I filter the Customers report by a custom date (one month), I get these PHP notices:
```
Notice | Undefined offset: 1535760000 /easy-digital-downloads/includes/admin/reporting/reports.php:2726
Notice | Undefined offset: 1 /easy-digital-downloads/includes/admin/reporting/reports.php:2726
Notice | Undefined offset: 0 /easy-digital-downloads/includes/reports/data/charts/v2/class-dataset.php:265
```

### Steps to reproduce the behavior
1. Visit Downloads > Reports > Customers.
2. I had to set the custom date range to a month long range which I knew should have data.

### Information (if a specific version is affected):
PHP Version: 7.4
EDD Version (or branch): main, release/3.1.0.2
WordPress Version: 6.1-RC2

Any other relevant information:
This has also been reported by a customer.

**Comment by cklosowski** [comment]
I only replicated this going on a 4 month range, in case anyone else is trying to.

**Comment by cklosowski** [comment]
@robincornett I think this was fixed by @zkawesome in his PR for #9547

**Comment by webzunft** [comment]
I just wanted to report the same issue. Happy to test a fix, if needed.
